rainier-sampler/src/main/scala/com/stripe/rainier/sampler/VarianceEstimator.scala

```diff
@@ -0,0 +1,61 @@
+package com.stripe.rainier.sampler
+
+class VarianceEstimator(size: Int) {
```

Comment:
For BFMI we only use this with size=1, but we'll want this to be multivariate later on for mass matrix adaptation.